src/deck.py

```diff
@@ -1,7 +1,20 @@
 from card import Card
-
+import random
+import uuid
 
 class Deck:
+    suits = ['Hearts', 'Diamonds', 'Clubs', 'Spades']
+    values = ['2', '3', '4', '5', '6', '7', '8', '9', '10', 'Jack', 'Queen', 'King', 'Ace']
+
     def __init__(self):
-        print("Deck constructor")
-        self._card = Card()
+        self.id = str(uuid.uuid4())
+        self.cards = [Card(suit, value) for suit in self.suits for value in self.values]
+        random.shuffle(self.cards)
+
+    def deal(self, num_cards):
+        if num_cards > len(self.cards):
+            return []  # or handle the case as needed
```

Comment:
What's with the comment here? :)

Do you want me to handle the case as needd? I think your Chat GPT is showing.